protocol/x/delaymsg/types/addresses.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+import authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+var (
+	ModuleAddress = authtypes.NewModuleAddress(ModuleName)
```

Comment:
```suggestion
	DelayMsgModuleAddress = authtypes.NewModuleAddress(ModuleName)
```

Reply:
I think that the current name makes sense, because the namespace of the package implies delaymsg, and this follows the same naming pattern we use with `ModuleName`... but not religious about this, still fine to change if you feel strongly about it.